fabric-enchantments-v0/build.gradle

```diff
@@ -0,0 +1,2 @@
+archivesBaseName = "fabric-enchantments-v0"
+version = getSubprojectVersion(project, "v0.0.0")
```

Comment:
>v

We start at 0.1.0, and without a v prefix. Just like all the other subprojects.

Reply:
Versioning is hopefully ok now.